test/parallel/test-console.js

```diff
@@ -76,17 +102,33 @@ assert.equal('foo bar\n', strings.shift());
 assert.equal('foo bar hop\n', strings.shift());
 assert.equal("{ slashes: '\\\\\\\\' }\n", strings.shift());
 assert.equal('inspect\n', strings.shift());
+assert.equal('foo\n', strings.shift());
+assert.equal('foo bar\n', strings.shift());
+assert.equal('foo bar hop\n', strings.shift());
+assert.equal("{ slashes: '\\\\\\\\' }\n", strings.shift());
+assert.equal('inspect\n', strings.shift());
+assert.equal('foo\n', errStrings.shift());
+assert.equal('foo bar\n', errStrings.shift());
+assert.equal('foo bar hop\n', errStrings.shift());
+assert.equal("{ slashes: '\\\\\\\\' }\n", errStrings.shift());
+assert.equal('inspect\n', errStrings.shift());
+assert.equal('foo\n', errStrings.shift());
+assert.equal('foo bar\n', errStrings.shift());
+assert.equal('foo bar hop\n', errStrings.shift());
+assert.equal("{ slashes: '\\\\\\\\' }\n", errStrings.shift());
+assert.equal('inspect\n', errStrings.shift());
 assert.equal("{ foo: 'bar', inspect: [Function] }\n", strings.shift());
 assert.equal("{ foo: 'bar', inspect: [Function] }\n", strings.shift());
 assert.notEqual(-1, strings.shift().indexOf('foo: [Object]'));
 assert.equal(-1, strings.shift().indexOf('baz'));
 assert.equal('Trace: This is a {"formatted":"trace"} 10 foo',
-             strings.shift().split('\n').shift());
```

Comment:
I think a change is warranted. The only reason it was in `strings` (i.e. `stdout`) is because the `console._stderr` was set to `process.stdout`. My change ensures we're testing both streams separately, and `trace` only outputs on `stderr`.